decompy/cli_frontend/ArgumentProcessor.py

```diff
@@ -0,0 +1,40 @@
+# from cli_frontend import LiftSubprocess as ls
```

Comment:
short names like this are annoying to read when you import from the bottom level IMO